src/internal/agent/hooks/flux-ocirepo.go

```diff
@@ -21,6 +22,11 @@ import (
 	v1 "k8s.io/api/admission/v1"
 )
 
+const (
+	HelmMediaTypeManifest = "application/vnd.cncf.helm.config.v1+json"
+	WebhookTimeout        = 10 * time.Second
```

Comment:
nit: This isn't the WebHookTimeout. Maybe registryFetchTimeout would fit better?